src/main/java/com/google/graphgeckos/dashboard/api/components/BuildInfo.java

```diff
@@ -0,0 +1,69 @@
+package com.google.graphgeckos.dashboard.api.components;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Represents the response json with the aggregated buildbot data.
+ */
+public class BuildInfo {
+
+  // git commit hash
+  private final String commitHash;
+  // time of git commit push
+  private final Date timestamp;
+  // link to to the repository
+  private final String repository;
+  // name of the branch
+  private final String branch;
+
+  // builders
+  private final List<Builder> builders;
+
+  public BuildInfo(String commitHash, Date timestamp, String repository,
+                   String branch, List<Builder> builders) {
+    this.commitHash = commitHash;
+    this.timestamp = timestamp;
+    this.repository = repository;
+    this.branch = branch;
+    this.builders = new ArrayList<>(builders);
+  }
+
+
+  /**
+   * Returns git commit hash of the tested changes commit.
```

Comment:
nit: capitalize git here (Git); ensure you use consistent capitalization across the file.